### PR TITLE
project file added and commited from branch

### PR DESCRIPTION
**Pull request by eyoab01**
in the project index.html,style.css and readme files are added and updated.

### CODE REVIEW

**Comment by KDlamini** [comment]
## Status: Invalid 🔴 

### Hi @eyoab01  🙋‍♀️,  

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request
 
You have submitted a project, that does not have a Pull Request description. Kindly write a summary of the features that you implemented in this milestone.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <menu>
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </menu>
+          <a class="btn-action">See Project</a>
+        </div>
```
Comment (thecodechaser) [inline index.html:82]: - [ ] Please reduce the left-right margin from the project card and make sure it matches [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) the design.

### Your design

image

### Figma design

image
****

File: style.css [hunk 2]
```diff
@@ -0,0 +1,535 @@
...
+  flex-direction: column;
+  max-width: 330px;
+  background: #fff;
+  border: 2px solid #dfe1e6;
+  border-radius: 16px;
+  box-shadow: 0 0 0.5px 0.25px inset #c8c6c6;
+  padding: 16px 16px 30px 16px;
+  margin-bottom: 90px;
+  transition: all ease-in-out 0.2s;
+}
```
Comment (thecodechaser) [inline style.css]: - [ ] Good job using the `FlexBox` for project cards. Please use the `CSS grid` for the project's main container as requested in the project requirements.
image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @eyoab01👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by eyoab01** [comment]
portfolio project all sections added.

**Comment by eyoab01** [comment]
Got it, thanks!

**Comment by eyoab01** [comment]
I will change it, but  I kindly request you to consider  The deadline
approaching. Can you finish for me please?
Thank you,

File: style.css (hunk 2 above)
Comment (Nemwel-Boniface) [inline style.css]: - [ ] As requested by the [previous reviewer](https://github.com/eyoab01/mobile-portfolio-project-final/pull/1#discussion_r956614232), kindly ensure that you use CSS grid to layout the elements in the main container.

image

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @eyoab01      :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Great job addressing some of the changes that were requested by a previous reviewer :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 3]
```diff
@@ -0,0 +1,535 @@
...
+.socials :nth-child(1) {
+  padding-left: 0;
+}
+
+#projects {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 142px 142px;
+}
```
Comment (Santiago220991) [inline style.css]: As requested by the last code reviewer kindly ensure that you use CSS grid to layout the elements in your #projects container. This is part of the requirements of the project. :scroll:

**Review by Santiago220991 — CHANGES_REQUESTED** [review]
Hi @eyoab01 , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!




## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by eyoab01** [comment]
Thanks for the feedback!

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️

Hi @eyoab01 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages


 ## Required Changes ♻️

### As requested by the last code reviewer kindly ensure that you use CSS grid to layout the elements in your #projects container. This is part of the requirements of the project. 📜

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — APPROVED** [review]
#   *Status: Approved* :tada: 

Hi @eyoab01, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

-  No linter errors. :heavy_check_mark: 
-  Correct GitHub flow. :heavy_check_mark: 
-   Good commit messages. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**